bin/verify-links.sh

```diff
@@ -134,24 +134,46 @@ for file in ${mdFiles}; do
       continue
     fi
 
-    # Local file href - skip for now.
-    # TODO add support for checking these
-    if [ "${ref:0:1}" == "#" ]; then
-      ref=${ref:1}
-      if ! grep "^$ref$" ${tmp}anchors1 > /dev/null 2>&1 ; then
-        echo $file: Can\'t find anchor \'\#${ref}\' | tee -a ${tmp}3
+    # Local repo markdown link (i.e. ref contains a #) 
+    if [[ "${ref/\#}" != "${ref}" ]]; then
```

Comment:
One cool thing about this work is that I'm learning (or remembering) way more about sh/bash stuff than I would normally - e.g. all of this variable expansion/substitution stuff.